spikesorters/basesorter.py

```diff
@@ -57,6 +58,9 @@ def __init__(self, recording=None, output_folder=None, verbose=False,
         self.parallel = parallel
         self.params = self.default_params()
 
+        if self.parallel:
+            assert self.have_parallel_thread, 'this sorter do not accept parallel=True for groups'
```

Comment:
Instead of raising I would print a warning and set parallel to False

Reply:
No I think it have to fail because it do not work.
And I don't this we should create a custom error for this.

Reply:
it is a feature that some sorter don't have so fail, no ?